ow_plexil/src/plans/owlat_plans/owlat-interface.h

```diff
@@ -2,87 +2,88 @@
 // Research and Simulation can be found in README.md in the root directory of
 // this repository.
 
+// PLEXIL interface to lander that is OWLAT-specific.
+
 #ifndef Owlat_Plan_Interface_H
 #define Owlat_Plan_Interface_H
 
-// PLEXIL interface to lander: commands, lookups, library plans, PLEXIL utilities
+#include "../common/common-interface.h"
 #include "owlat-lander-commands.h"
 
-// Utility commands; issue ROS_INFO, ROS_WARN, and ROS_ERROR, respectively.
-Command log_info (...);
-Command log_warning (...);
-Command log_error (...);
+// Note that in cases where OceanWATERS has an action with the same
+// name (but different signature, else it would be in ../common), the
+// one here is prefixed "Owlat".
 
```

Comment:
Or maybe another name since the `TOOL` prefix is redundant to `TOOL_FRAME`.

I like common prefixes that reflect the category of the constants, so for the frame constants I would recommend `FRAME_BASE` and `FRAME_TOOL`.

Reply:
Good suggestion, done.  And I added a SetTool in a few plans where it was missing.